src/components/section/section.md

```diff
@@ -0,0 +1,41 @@
+Sections can be used to semantically group controls, information, and inputs. Sections get the `region` ARIA role by default, and the `title` prop is required to support accessibility with a contained heading. For now, the section component should only be used in Custom Apps, while the "page layout" spec is still in flux.
+
+```
+<Section
+  title="I am a section"
+  description="Test description"
+/>
+```
+
+Sections can contain child elements, like any other element:
```

Comment:
Do we have an example of a Section with my rows of fields?

Reply:
Every direct child is a row, so in this case each input is a row. Tables can be achieved by using a container with the proper CSS. Is that something we want to exemplify here?

Reply:
Right, I didn't realized it was a row per field. I think what I want to see as well is a row with many fields.